minetest-mods/unified_inventory
# [NO SQUASH] Use player meta for waypoints

This PR migrates old `datastorage` entries to player metadata. Includes table format documentation and some code cleanups.

Translation files updated using https://github.com/minetest-tools/update_translations

How to test:

 * Use master, and enable https://github.com/minetest-technic/datastorage
 * Set up some waypoints
 * Apply PR
 * Check whether all data is preserved, and that the data storage file is (almost) truncated. The datastorage API does not delete files when the data is empty, presumably thought as a security mechanism.

If there are no objections or remarks, I will merge this PR in a few days.
